libraries/commerce/product/subscriptions/index.js

```diff
@@ -47,6 +50,12 @@ function product(subscribe) {
       dispatch(setProductVariantId(null));
     }
   });
+
+  subscribe(productRelationsReceived$, ({ dispatch, getState, action }) => {
```

Comment:
Could you please add a test for your new subscription? You don't have to test the ones which are already there.

Reply:
Done.